### ISSUE TITLE
BGE-M3预训练问题

### ISSUE THREAD

**Issue by chengzi-big** [FlagOpen/FlagEmbedding#817]
请问，关于BGE-M3模型的所有预训练代码都开放了吗？

**Comment by staoxiao** [comment]
Yes, the retromae script is at https://github.com/FlagOpen/FlagEmbedding/tree/master/examples/pretrain, the fine-tuning script is at https://github.com/FlagOpen/FlagEmbedding/tree/master/examples/unified_finetune
